story-generator/src/app/app.css

```
@@ -434,172 +508,180 @@ input[type='range']::-webkit-slider-thumb {
+  background: var(--paper-strong);
+  color: var(--ink);
+  line-height: 1.72;
+  padding: 24px;
+  font-family: Georgia, "Times New Roman", serif;
+  font-size: 1.06rem;
 }
 
-.story-header h2 {
-  font-size: 2.2rem;
-  line-height: 1.05;
+:host ::ng-deep .chapter-content p {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)